engine/src/main/scala/cromwell/jobstore/SqlJobStore.scala

```diff
@@ -56,6 +56,7 @@ class SqlJobStore(sqlDatabase: JobStoreSqlDatabase) extends JobStore {
       case "Int" => WdlIntegerType
       case "Float" => WdlFloatType
       case "Boolean" => WdlBooleanType
+      case "File" => WdlFileType
```

Comment:
We should tech talk this but long story short: you'll never have a WdlGlobFile here. Always reinstantiate to WdlSingleFile